### PR TITLE
minor corections in cass_koopmans.rst

### PR DESCRIPTION
**Pull request by AnjuJoon**

### CODE REVIEW

File: source/rst/cass_koopmans.rst [hunk 1]
```diff
@@ -35,7 +35,7 @@ The lecture uses important ideas including
 
 -  A min-max problem for solving a planning problem
 
--  A **shooting algorithm** for solving difference equations subject
+-  A **shooting algorithm** for solving difference equations subjected
```
Comment (jstac) [inline source/rst/cass_koopmans.rst]: please undo

File: source/rst/cass_koopmans.rst [hunk 2]
```diff
@@ -106,15 +106,15 @@ A representative household is endowed with one unit of labor at each
 :math:`t` and likes the consumption good at each :math:`t`
 
 The representative household inelastically supplies a single unit of
-labor :math:`N_t` at each :math:`t`, so that
+labor :math:`N_t` at each :math:`t` so that
```
Comment (jstac) [inline source/rst/cass_koopmans.rst]: please undo

File: source/rst/cass_koopmans.rst [hunk 3]
```diff
@@ -290,48 +290,48 @@ feasibility constraints
...
-  
-  \begin{align} C_{t+1} =\left(\beta C_t^{\gamma}[f'(K_{t+1}) + 
+.. math::
+
+  \begin{align} C_{t+1} =\left(\beta C_t^{\gamma}[f'(K_{t+1}) +
   (1-\delta)]\right)^{1/\gamma} \notag\\= C_t\left(\beta [f'(K_{t+1}) +
   (1-\delta)]\right)^{1/\gamma} \end{align}
 
-The above first-order condition for consumption is called an **Euler
+The above first-order condition for consumption is called a **Euler
```
Comment (jstac) [inline source/rst/cass_koopmans.rst]: please undo

File: source/rst/cass_koopmans.rst [hunk 4]
```diff
@@ -290,48 +290,48 @@ feasibility constraints
...
 equation**
 
 It tells us how consumption in adjacent periods are optimally related
-to each other and to capital next period
+to each other and to the capital next period
```
Comment (jstac) [inline source/rst/cass_koopmans.rst]: please undo

File: source/rst/cass_koopmans.rst [hunk 5]
```diff
@@ -618,7 +618,7 @@ Now we can plot
             ax.plot(path)
             ax.set(ylabel=y, title=title, xlabel='t')
 
-        # Plot steady state value of capital
+        # Plot steady-state value of capital
```
Comment (jstac) [inline source/rst/cass_koopmans.rst]: please remove the hyphen in all instances of "steady-state" and "Steady-state" .

It's not standard in economics.  For example, see https://en.wikipedia.org/wiki/Golden_Rule_savings_rate

File: source/rst/cass_koopmans.rst [hunk 6]
```diff
@@ -1069,7 +1069,7 @@ firm makes positive profits on each additional unit of
 arbitrarily large
 
 But setting :math:`\tilde k_t = + \infty` is not physically feasible,
-so presumably **equilibrium** prices will assume values that present
+so presumably, **equilibrium** prices will assume values that present
```
Comment (jstac) [inline source/rst/cass_koopmans.rst]: either "so, presumably," or no commas please.

File: source/rst/cass_koopmans.rst [hunk 7]
```diff
@@ -629,7 +629,7 @@ Now we can plot
     plot_paths(paths)
 
 
-Setting :math:`K_0` Equal to Steady State :math:`K`
+Setting :math:`K_0` Equal to steady state :math:`K`
```
Comment (jstac) [inline source/rst/cass_koopmans.rst]: Shouldn't these be capitals?

Comment (AnjuJoon) [reply source/rst/cass_koopmans.rst]: Ohh this is a silly mistake. Thanks @jstac

**Pull request merged**